kairon/shared/models.py

```
@@ -99,6 +99,7 @@ class LlmPromptSource(str, Enum):
     action = "action"
     history = "history"
     tag = "tag"
+    crud= "crud"
```

Flake8 (7.2.0)

[error] missing whitespace around operator

(E225)